server/src/entrypoint.sh

```diff
@@ -7,3 +7,9 @@ ln -s /srv/cvmfs /var/www/cvmfs
 a2enmod headers expires proxy proxy_http
 service apache2 start
 
+# Add cvmfs_server resign command
+cvmfs_server resign
+
+# Schedule cvmfs_server resign command to run daily using a cron job
+echo "0 0 * * * root cvmfs_server resign" > /etc/cron.d/cvmfs_resign
+cron
```

Comment:
Say that this file is not being run, perhaps we should implement this functionality in python? 

The current entrypoint that we use for this container is https://github.com/WATonomous/infra-config/pull/3327/files#diff-71f2d12c2105449a857636ad2b097b5e4ad65d3d0b82b9267c587daaf8aad8c9R71

I do like the simplicity of this cron approach though. Perhaps we can combine them somehow?

Reply:
I think python would actually be easier, doesn't seem like cron is actually installed in the container.

alexboden@trpro-slurm1:~$ docker run -it ghcr.io/watonomous/cvmfs-ephemeral:sha-661033b65e0c3b4546bbece816095d438a55c1c9-server sh
# which cron
# which crontab